zipkin-server/src/main/java/zipkin2/server/internal/InternalZipkinConfiguration.java

```diff
@@ -30,7 +30,7 @@
 import zipkin2.server.internal.scribe.ZipkinScribeCollectorConfiguration;
 import zipkin2.server.internal.ui.ZipkinUiConfiguration;
 
-@Configuration
+@Configuration(proxyBeanMethods=false)
```

Comment:
It seems that this is equivalent to removing the annotation

https://docs.spring.io/spring-boot/docs/2.2.0.BUILD-SNAPSHOT/api/org/springframework/boot/SpringBootConfiguration.html

I think an annotations (can be `@Component` according to that doc) is needed for autoconfiguration but since we don't use autoconfig, we may be able to get away with not adding the annotation at all.

Reply:
interesting.. yeah when I removed auto-configuration, I had to undo the proxies anyway..

lemme try w/o annotations!

Reply:
there were some proxying and magic left in BeanPostProcessors.. I've attempted to delete those also in the latest push. Will need to check the variants before merge or before release.